mds/management/commands/poll_providers.py

```diff
@@ -114,6 +114,7 @@ def poll_status_changes(self, provider):
                                     "lat": event_location["geometry"]["coordinates"][1],
                                     # XXX altitude, etc.?
                                 },
+                                "battery_pct": status_change["battery_pct"],
```

Comment:
`.get`. maybe ? 😋

Reply:
true in the spec the field is indeed optional.